CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 ## 1.1.0-beta.2 (Unreleased)
 
+### New Features
+
+
```

Comment:
Presumably this is for Pnp?

Reply:
yea so this is going into master. I will then port this change to the pnp feature branch and then add the pnp feature set to that section